src/D2L.Bmx/Okta/Models/OktaError.cs

```diff
@@ -0,0 +1,8 @@
+namespace D2L.Bmx.Okta.Models;
+
+internal record OktaError(
+	string ErrorCode,
+	string ErrorSumamry,
```

Comment:
```suggestion
	string ErrorSummary,
```

Reply:
EDIT: lets delete OktaError altogether, we can add anything necessary for handling Okta API errors when we get to that.